.travis.yml

```diff
@@ -5,6 +5,7 @@ env:
         - DATABASE_USER=travis
         - DATABASE_PASSWORD=travis
         - DJANGO_CURRENT_VERSION=1.8.12
+        - RALPH2_RALPH3_CROSS_VALIDATION_ENABLED=1
```

Comment:
I add because of this https://github.com/allegro/ralph/pull/2543/files#diff-fad12832045cc87fe41573929b84d835R67